hazelcast/src/main/java/com/hazelcast/map/impl/operation/EntryOffloadableSetUnlockOperation.java

```diff
@@ -151,18 +155,27 @@ protected void writeInternal(ObjectDataOutput out) throws IOException {
         UUIDSerializationUtil.writeUUID(out, caller);
         out.writeLong(begin);
         out.writeObject(entryBackupProcessor);
+        // RU_COMPAT_4_0
+        if (out.getVersion().isGreaterOrEqual(Versions.V4_1)) {
```

Comment:
Afaik `EntryOffloadableSetUnlockOperation` must be `Versioned` to get version of the stream in EE.

Reply:
fixed